docker/config.sh

```diff
@@ -26,9 +26,9 @@ function install_gdal {
 
     echo "installing gdal from source" && (
         cd ~
-        wget download.osgeo.org/gdal/3.8.3/gdal383.zip
-        unzip gdal383.zip
-        cd gdal-3.8.3
+        wget https://github.com/fvankrieken/gdal/archive/refs/heads/3.8-revert-null-date-behavior.zip
```

Comment:
just wondering how this zip/link is generated. I thought maybe it was a subfolder or a [tag](https://github.com/fvankrieken/gdal/tags) in your fork but I can't find anything named archive

Reply:
It is that